Sources/TuistCore/Graph/GraphTraverser.swift

```diff
@@ -176,11 +177,24 @@ public class GraphTraverser: GraphTraversing {
 
         let bundles = filterDependencies(
             from: .target(name: name, path: path),
-            test: isDependencyResourceBundle,
+            test: { dependency in
+                isDependencyResourceBundle(dependency: dependency) && !isDependencyExternal(dependency)
+            },
             skip: canHostResources
         )
+        let externalBundles = filterDependencies(
```

Comment:
What about adding a documentation comment here? Alternatively, they can use the Git history to gain the context, but I thought saving that indirection would be useful when looking at this code in the future.

Reply:
Agree, added 👍